packages/e2e-test-app/index.ts

```diff
@@ -0,0 +1,23 @@
+import * as childProcess from 'child_process';
+import { startWebServer, killWebServer } from './src/mock-web-server';
+
+startWebServer();
+
+const testringProcess = childProcess.exec(
+    './node_modules/.bin/testring run --config ./test/selenium/config.js',
+    {},
+    (error, stdout, stderr) => {
+        killWebServer();
```

Comment:
There is no error handling, so even if tests are failed the task won't show this